Documentation/src/for-each.md

```diff
@@ -0,0 +1,19 @@
+# `IEnumerable` extension method `.ForEach`
+With the `.ForEach` extension method, you can invoke an action for each item in an enumerable, just like a `for each` statement would allow you to do.

```

Comment:
The `foreach` keyword is one word
```suggestion
With the `.ForEach` extension method, you can invoke an action for each item in an enumerable, just like a `foreach` statement would allow you to do.

```